daemon/user/daemon.go

```diff
@@ -0,0 +1,138 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package user
+
+import (
+	"bytes"
+
+	"github.com/godbus/dbus"
+
+	"fmt"
+
+	"github.com/godbus/dbus/introspect"
+	tomb "gopkg.in/tomb.v2"
+)
+
+const (
+	busName  = "com.canonical.SafeLauncher"
```

Comment:
Nitpick: I know this is what we currently implement but I'd like us to listen on both `io.snapcraft.SomethingNice` and `com.canonical.SafeLauncher` (the second one can be deprecated later)